unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarHUDController.cs

```diff
@@ -551,8 +551,10 @@ public void ShowTutorialOption(bool isActive)
             view.moreMenu.ShowTutorialButton(isActive);
     }
 
-    private void SceneController_OnNewPortableExperienceSceneAdded(GlobalScene newPortableExperienceScene)
+    private void SceneController_OnNewPortableExperienceSceneAdded(IParcelScene scene)
     {
+        GlobalScene newPortableExperienceScene = (GlobalScene)scene;
```

Comment:
I'd add some checks here:

`if(!scene is GlobalScene newPortableExperienceScene) return;`

Reply:
On the other hand something feels out of place here (same in LoadingFeedbackController).

I find strange that the listener to the `OnNewPortableExperienceSceneAdded` event has to know that the `IParcelScene` provided is also a `GlobalScene`. Maybe we should have an `IGlobalScene` as well as part of our WorldRuntime interfaces. 

In any case this issue can be addressed in other PR, if you agree with the concern, of course.

Reply:
Yes, I think this issue should be addressed on another PR. The code already relied on the `GlobalScene` cast to succeed and wasn't doing any checks, so the code would fail in any case. I saw that and realized that just using static casting would have the same behaviour, that's the reasoning that led me to the change.

Reply:
Added the additional check just in case.